src/inmanta/moduletool.py

```diff
@@ -1052,7 +1053,5 @@ def get_setup_cfg(self, config_in: Optional[configparser.ConfigParser] = None) -
         config["options"]["zip_safe"] = "False"
         config["options"]["include_package_data"] = "True"
         config["options"]["packages"] = "find_namespace:"
-        config.add_section("options.package_data")
-        config["options.package_data"]["*"] = "files/*, model/*, templates/*, setup.cfg"
```

Comment:
We should also make this change in the inmanta-module-template on the branch for modules v2